apps/bynder/src/index.js

```diff
@@ -5,6 +5,8 @@ import logo from './logo.svg';
 
 const CTA = 'Select a file on Bynder';
 
+// DOCS POINT TO THIS
+// const BYNDER_BASE_URL = 'https://ucv.bynder.com';
 const BYNDER_BASE_URL = 'https://d8ejoa1fys2rk.cloudfront.net';
```

Comment:
The documentation points to the commented out `BYNDER_BASE_URL`, is there any reason known to the direct CDN reference?

Reply:
any thoughts here @jsdalton?